src/helm/benchmark/presentation/run_specs_lm_opinions_ai21_steer-bio.conf

```diff
@@ -0,0 +1,5 @@
+entries: [
+{description: "opinions_qa:survey_type=Pew_American_Trends_Panel_disagreement_500,context=steer-bio,num_logprobs=10,model=ai21/j1-jumbo,num_train_trials=1", priority: 1}
```

Comment:
You define the RunExpander so that you can just write one line with `model=ai21`, which expands into all the ai21 models.

Reply:
👍